<?xml version="1.0" ?>
<launch>
	<node pkg="reconfigure" name="cheetah_pos_reconfigure" type="pos_reconfigure_server.py" output="screen" />

	<node name="rqt_gui" pkg="rqt_gui" type="rqt_gui" args="-s reconfigure"/>

	<node pkg="reconfigure" name="cheetah_pos_pub_rqt" type="pos_pub.py" output="screen"/>
</launch>